src/land/ascat_ssm2ioda.py

```diff
@@ -65,14 +65,18 @@ def _read(self):
             self.varAttrs[iodavar, iconv.OvalName()]['coordinates'] = 'longitude latitude'
             self.varAttrs[iodavar, iconv.OerrName()]['coordinates'] = 'longitude latitude'
             self.varAttrs[iodavar, iconv.OqcName()]['coordinates'] = 'longitude latitude'
-            self.varAttrs[iodavar, iconv.OvalName()]['units'] = 'percent'
-            self.varAttrs[iodavar, iconv.OerrName()]['units'] = 'percent'
-            self.varAttrs[iodavar, iconv.OqcName()]['units'] = 'unitless'
+            # Per ioda conventions soilMoistureNormalized has 1 unit (ie percent/100) (2022/03/09)
+            self.varAttrs[iodavar, iconv.OvalName()]['units'] = '1'
+            # Per ioda conventions soilMoistureNormalized has 1 unit (ie percent/100) (2022/03/09)
+            self.varAttrs[iodavar, iconv.OerrName()]['units'] = '1'
+            # Per ioda conventions PreQC fields come without unit (2022/03/09)
+#           self.varAttrs[iodavar, iconv.OqcName()]['units'] = '1'
```

Comment:
Remove commented out line (and above line probably?)

Reply:
This has been resolved by @gthompsnJCSDA. Please, see the commits below.